test/scylla-scripts-pytest/scylla_product.py

```diff
@@ -0,0 +1,2 @@
+# dummy file for testing
+PRODUCT='scylla'
```

Comment:
What about scylla-enterprise?

Reply:
for now it's not covering it, not that I'm aware of any logic based on that. it's maybe for finding the paths to directories.

if it would be needed, this file can be removed and created inside the tests as needed.